### PR TITLE
Responsive Navigation bar added

### PR DESCRIPTION
**Pull request by sheetalj2205**
## Related Issue
Issue #303 
- Info about the related issue
The header didn't have a navigation bar.
Closes: #[303]

### Describe the changes you've made
I have added the navigation bar in the header.
Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [x
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [ 
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

**Comment by pinkman7009** [comment]
Add the correct issue number and make a PR again. Also your code seems to have many file changes which aren't necessary.

File: src/components/Nav/Burger.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import RightNavbar from './RightNavbar';
+
+const StyledBurger = styled.div`
+  width: 2rem;
```
Comment (Anurag5086) [inline src/components/Nav/Burger.jsx:7]: Please Make a separate css file for all the styles.

File: src/components/Nav/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Burger from './Burger';
+
+const Nav = styled.nav`
```
Comment (Anurag5086) [inline src/components/Nav/Navbar.jsx:6]: Make css file.

File: src/components/Nav/RightNavbar.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const Ul = styled.ul`
```
Comment (Anurag5086) [inline src/components/Nav/RightNavbar.jsx:6]: Make css file

**Review by Anurag5086 — CHANGES_REQUESTED** [review]
Make a separate css file for styles.

**Comment by msk4862** [comment]
@sheetalj2205 Please create PR with your commits only and add screenshots as well.
> Before pushing your commit, pull the changes from `remote` and push it  in your local repo to synchronise your local repo with remote.

File: src/components/Nav/RightNavbar.jsx (hunk 3 above)
Comment (pinkman7009) [reply src/components/Nav/RightNavbar.jsx:6]: She has used styled components for this cause there is a functionality of the open prop. So I don't think an scss file can be created.

**Pull request closed without merging**